man/systemd.unit.xml

```diff
@@ -1031,6 +1033,15 @@
         whether a certain path exists, is a regular file and marked
         executable.</para>
 
+        <para><varname>ConditionUID=</varname> takes a numeric
+        <literal>UID</literal>. This condition may be used to check
+        wether the service manager is running as the given user id.
+        This option is only meaningful for the user managers.</para>
+
+        <para><varname>ConditionGID=</varname> is similar
+        to <varname>ConditionUID=</varname> but verifies that the
+        service manager is in the specified group.</para>
```

Comment:
I'd say this a bit differently: users are members of groups, not managers/processes. Hence I would say: `… but verifies that the service manager's real or effective group ID, or any of its auxiliary group IDs match the specified group`, or something like that...